devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1558,6 +1673,27 @@ bool HumanStateProvider::impl::applySecondaryCalibration(
     return true;
 }
 
+bool HumanStateProvider::impl::applyFixedTransformForInputData(std::unordered_map<std::string, iDynTree::Transform> &transforms_in)
```

Comment:
what do you think about renaming this to `applyFixedRightRotationForInputTransformData()` ?

Reply:
Sure, I can do it. I would also like to rename all the methods/commands adding `right`/`left` rotation keyworkds, but probably in a future PR.